libcob/coblocal.h

```diff
@@ -300,6 +300,7 @@ typedef struct __cob_settings {
 
 	/* call.c */
 	int	cob_physical_cancel;	/* 0 "= "logical only" (default), 1 "also unload", -1 "never unload" */
+	unsigned int	cob_load_global; /* Wether dlopen should use the global or local namespace */
```

Comment:
that's not only dlopen so something like `/* hint for dynamic linker to make symbols available 0=global, 1=local */` (maybe shorter)